Standards/scs-0005-v1-project-governance.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Sovereign Cloud Standards
+type: Procedural
+status: Draft
+track: Global
+stabilized_at:
+description: |
+  SCS-0005 outlines the structure and governance of the SCS community by the SCS Project Board and how this is elected.
+---
+
+## Introduction
+
+The [Sovereign Cloud Stack (SCS)](https://scs.community) provides standards
+for a range of cloud infrastructure types as well as a modular open-source
+reference implementation.
+The project is governed by the _SCS Project Board_.
+
+## Role of the _SCS Project Board_
+
+The role of the _SCS Project Board_ is the overall governance of the SCS Community and Project.
+This happens together with the _Forum SCS-Standards_ of the Open Source Business Alliance. To further
+underline this alignment, the _Forum SCS-Standards_ is part of the _SCS Project Board_.
+The _SCS Project Board_ itself is elected by the _SCS Community_.
+
+### Definitions
+
+#### _SCS Project_
+
+The _SCS Project_ is the Open-Source project that consists of the software, documentation, documents, blog posts as well as the people ("_SCS Community_") working on this.
+
+#### _SCS Community_
+
+The collective of people, companies, and organizations promoting the idea of the _SCS Project_ as well as the people working on the various aspects.
+
+## Election of the _SCS Project Board_
+
+### Term
+
+The _SCS Project Board_ is elected for the term of one year. Elections are done
+in November. The December serves as a transition period.
+
+### Seats on the board
+
+The _SCS Project Board_ contains five seats. One of these seats is filled by
+the delegate of the _Forum SCS-Standards_. The other four seats are voted upon.
+
+### Nominations
+
+Every person who is part of the Sovereign Cloud Stack GitHub organization can be
+nominated for the board. Likewise, one can nominate oneself.
+
+### Eligible for voting
+
+Every person who is a member of the GitHub organization "Sovereign Cloud Stack" is eligible for voting.
+
+### Electoral management
+
+The voting process is governed by the _Forum SCS-Standards_.
+
+### Mechanisms
+
+Each voting person has eight (8) points that can be cast during the voting process.
+The points can be distributed between four candidates.
```

Comment:
@berendt this mechanism came from you. Please elaborate and clarify directly in the file.

Reply:
From my understanding "the points can be distributed between four candidates" means: yes, you can also allocate all of them on one candidate. Wether or not this is what @berendt had in mind, when he suggested this: 🤷🏻‍♂️ 

I was the one who suggested using heliosvoting.org for this, since that is a system well-known to work (we've previously used it at the OSBA for votings). However heliosvoting.org does not support this point-based voting (eg. one of course can enter the points into a textfield, but the system does not technically assist in wether this done correctly or not).

@berendt do you have a suggestion on how this could be done in a way that you envisioned?

Reply:
I brought this point up, because I think the voting mechanism is the most important part of the document and it not being clearly specified might lead to unwanted outcomes in the future.
We can absolutely not clarify further on "how to distribute" and just accept any way that is within the bounds. We should however still clarify how the actual voting process is done. Is it a secret/public voting? Is it done in the same document that handles nominations or via third party tool (there is probably more open questions that I can not immediately think about)?
Should we not specify anymore, the "specification work" and the "how are we doing this" is just put upon the shoulders of the current and future forum members (assuming this is what you meant with "The voting process is governed by the Forum SCS-Standards.").
Would be nice to get your opinion on this @berendt :)

Reply:
Very interesting questions! We should use different pages/tools for nominations and voting. So, we have the challenges that the tool should allow the point giving voting, the results should be anonymously, but it must also ensure that a person can only vote once... 

What needs to be more specific? @michaelbayr